backend/routes/issue.js

```diff
@@ -242,6 +243,26 @@ function renderIssuesHTML(req, res, next) {
 	let findIssue;
 	const noClean = false;
 
+	/**
+	* Create dynamic Print report values to use in template.
+	**/
+	const reportValues = {};
+	const reportDate = moment().format("Do MMMM YYYY");
```

Comment:
Make a function to format date (can be within this file for now), so if we end up changing the date formatting we only have to change it in one place.